# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libtorrent/configure-ldflags-fix.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- libtorrent-rasterbar-0.14.6/configure.in.orig	2009-11-10 19:04:06.000000000 +0100
+++ libtorrent-rasterbar-0.14.6/configure.in	2009-11-10 19:05:06.000000000 +0100
@@ -157,7 +157,6 @@

 dnl Apply boost config.
 CPPFLAGS="$CPPFLAGS $BOOST_CPPFLAGS"
-LDFLAGS="$LDFLAGS $BOOST_LDFLAGS"

 if [[ "x$BUILD_TESTCLIENT" != "xno" ]]; then
 	CLIENT_TEST_BIN=client_test;
